DirectEmployers/MyJobs
# Correct merge issues in reporting

## What's this PR do?
Make reporting work again
## Screenshots (if appropriate)

## Where should the reviewer start?
Reporting
## How should this be manually tested?
Head to reporting.

Make sure it works.

Yup. It was totally broken for a second. So.. Just test the whole app :)
## Any background context you want to provide?
Some old code was reintroduced due to some merge-clobbering. This PR should correct those issues.
